ci/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -ex
+
+cargo build --release --target $TARGET
+cargo build --release -p multirust-dist --target $TARGET
+
+if [ -z "$SKIP_TESTS" ]; then
+  cargo test --release --target $TARGET
+fi
```

Comment:
Why does this run `cargo build --release -p multirust-dist`? `multirust-dist` has tests that should be run, so I expect to see `cargo test --release -p multirust-dist`.

Also if this is building every build with `--target` then that affects deployment. Probably the prepare-deploy-travis script needs to change to copy the files from the target-specific directory. It looks though like the docker config is overwriting `CARGO_TARGET_DIR` only on linux, so the artifacts are going to be in different places depending on platform.